libs/partners/huggingface/langchain_huggingface/embeddings/huggingface_endpoint.py

```diff
@@ -46,11 +46,15 @@ class Config:
 
         extra = Extra.forbid
 
-    @root_validator()
+    @root_validator(pre=False, skip_on_failure=True)
```

Comment:
This should be pre=True based on the logic below -- all of it can be done as pre-init logic

Just requires re-writing values['model'] to values.get('model')

Reply:
Merging as is, but if you can please re-write this with `pre=True`